lib/vm/src/memory.rs

```diff
@@ -116,6 +116,27 @@ impl WasmMmap {
 
         Ok(prev_pages)
     }
+
+    /// Copies the memory
+    /// (in this case it performs a copy-on-write to save memory)
+    pub fn fork(&mut self) -> Result<Self, MemoryError> {
```

Comment:
Let's use clone. I think is a more accurate naming

Reply:
Renamed to `duplicate()`.